temp/static_analysis.py

```diff
@@ -0,0 +1,340 @@
+static_simulation.geometries.append(line)
+# static_simulation.plot(mesh=True, geometry=True, boundary_conditions=True)
+
+
+# -----------------------------------------------------------------------------------------
+# Define and use boundary conditions
+# TODO: TBD
+
+
+# -----------------------------------------------------------------------------------------
+# Define and use loads
+# TODO: TBD
+
+# Loads are not read from files yet
+# A load would then be derived from a DataObject, which has a mesh support, thus:
+# load = Load(data=data_object)
+# Mapping loads from one mesh to another (which is the same as DataObject.interpolate?)
+# load.interpolate(mesh=another_mesh)
+# This means a load has a support, either nodal, elemental or face.
+# This means location.face must be available.
+# It can also vary with step/time/frequency/rpm or any dimension present in the DataObject.
+
+# The metadata and functionalities unique to a Load would be:
+# - its naming
+# - its string representation
+# - its graphic representation
+
+# -----------------------------------------------------------------------------------------
+# Define and use a selection
+selection = Selection()
+selection.select_nodes(nodes=[1, 2, 3])
+selection.select_elements(elements=[1, 2, 3])
+selection.select_time_freq_indices(time_freq_indices=[1])  # Rework?
+# selection.select_steps(steps=[1])
+# selection.select_geometry(geometry=Line())
+selection.select_named_selection(named_selection=named_selections[0])
+
+# Intersect two spatial selections
+selection_1 = Selection()
+selection_1.select_nodes(nodes=[1, 2, 3])
+selection_2 = Selection()
+selection_2.select_nodes(nodes=[3, 4, 5])
+
+space_sel_1 = selection_1.spatial_selection
+print(space_sel_1)
+# Spatial selection with 3 node(s):
+# [1, 2, 3]
+space_sel_2 = selection_2.spatial_selection
+print(space_sel_2)
+# Spatial selection with 3 node(s):
+# [3, 4, 5]
+
+# space_sel_1.intersect(space_sel_2)
+print(space_sel_1)
+# Spatial selection with 1 node(s):
+# [3]
+
+# Union of two spatial selections
+selection_3 = Selection()
+selection_3.select_nodes(nodes=[1, 2, 3])
+selection_4 = Selection()
+selection_4.select_nodes(nodes=[3, 4, 5])
+space_sel_3 = selection_3.spatial_selection
+space_sel_4 = selection_4.spatial_selection
+
+# space_sel_3.union(space_sel_4)
+# print(space_sel_3)
+# Spatial selection with 5 node(s):
+# [1, 2, 3, 4, 5]
+
+# Do we then need to explicitly update the spatial_selection of selection_1?
+
+# Select based on constructed geometry
+# TODO
+
+# Set a selection as default for a simulation for further result extractions
+static_simulation.activate_selection(selection=selection)
+
+# Remove a selection as default for a simulation for further result extractions
+static_simulation.deactivate_selection()
+
+# Apply a SpatialSelection to a Simulation to get resulting list of entities IDs (nodes or elements)
+list_of_IDs = selection.spatial_selection.apply_to(static_simulation)
```

Comment:
Same as my previous comment.

Reply:
I am not sure I see which previous comment you refer to